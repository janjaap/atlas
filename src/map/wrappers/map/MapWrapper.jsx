import React from 'react';
import { render } from 'react-dom';
import { Provider } from 'react-redux';

import MapContainer from '../../containers/map/MapContainer';

const MapWrapper = () => (
  <Provider store={window.reduxStore}>
    <MapContainer />
  </Provider>
);

export default MapWrapper;

window.React = window.React || React;
// istanbul ignore next
window.render = window.render || render;
window.MapWrapper = MapWrapper;
