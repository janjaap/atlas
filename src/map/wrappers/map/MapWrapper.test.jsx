import React from 'react';
import configureMockStore from 'redux-mock-store';
import { shallow } from 'enzyme';

import MapWrapper from './MapWrapper';

describe('MapWrapper', () => {
  it('should render', () => {
    const store = configureMockStore()({});
    global.reduxStore = store;
    const wrapper = shallow(<MapWrapper />, { context: { store } }).dive();

    expect(wrapper).toMatchSnapshot();
  });
});
