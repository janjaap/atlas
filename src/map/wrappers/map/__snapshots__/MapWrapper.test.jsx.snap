// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapWrapper should render 1`] = `<Connect(MapContainer) />`;
